<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dashboard</class>
 <widget class="QMainWindow" name="Dashboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>725</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dashboard</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true">QFrame {	
	background-color: rgb(56, 58, 89);	
	color: rgb(220, 220, 220);
	border-radius: 10px;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <widget class="QLCDNumber" name="lcdNumberCalories">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>480</y>
         <width>141</width>
         <height>71</height>
        </rect>
       </property>
      </widget>
      <widget class="QLCDNumber" name="lcdNumberSpeed">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>190</y>
         <width>141</width>
         <height>71</height>
        </rect>
       </property>
      </widget>
      <widget class="QLCDNumber" name="lcdNumberDistance">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>380</y>
         <width>141</width>
         <height>71</height>
        </rect>
       </property>
      </widget>
      <widget class="QLCDNumber" name="lcdNumberTime">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>80</y>
         <width>351</width>
         <height>71</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
      </widget>
      <widget class="QLCDNumber" name="lcdNumberSteps">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>280</y>
         <width>141</width>
         <height>71</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="labelGroupName">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>600</y>
         <width>471</width>
         <height>61</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(170, 0, 255);</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;Group 76 - Treadmill Dashboard&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_time">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>80</y>
         <width>141</width>
         <height>71</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color:rgb(85, 255, 255);</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:26pt; font-weight:600;&quot;&gt;Time&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_speed">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>190</y>
         <width>141</width>
         <height>71</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 170, 255);</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:20pt; font-weight:600;&quot;&gt;Speed&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_steps">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>280</y>
         <width>141</width>
         <height>71</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 170, 255);</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:20pt; font-weight:600;&quot;&gt;Steps&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_distance">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>380</y>
         <width>141</width>
         <height>71</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 170, 255);</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:20pt; font-weight:600;&quot;&gt;Distance&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_calories">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>480</y>
         <width>141</width>
         <height>71</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 170, 255);</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:20pt; font-weight:600;&quot;&gt;Calories&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBoxSpeed">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>210</y>
         <width>81</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="placeholderText">
        <string/>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBoxDistance">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>400</y>
         <width>81</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="placeholderText">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_cal">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>480</y>
         <width>51</width>
         <height>71</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:20pt; font-weight:600;&quot;&gt;cal&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
